styles/roblox/catppuccin.user.css

```diff
@@ -0,0 +1,199 @@
+/* ==UserStyle==
+@name Roblox Catppuccin
+@namespace github.com/catppuccin/userstyles/styles/roblox
+@homepageURL https://github.com/catppuccin/userstyles/tree/main/styles/roblox
+@version 1.0.0
+@updateURL https://github.com/catppuccin/userstyles/raw/main/styles/roblox/catppuccin.user.css
+@supportURL https://github.com/catppuccin/userstyles/issues?q=is%3Aopen+is%3Aissue+label%3Aroblox
+@description Soothing pastel theme for roblox.com
+@author Catppuccin
+@license MIT
+
+@preprocessor less
+@var select lightFlavor "Light Flavor" ["latte:Latte*", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha"]
+@var select darkFlavor "Dark Flavor" ["latte:Latte", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha*"]
+@var select accentColor "Accent" ["rosewater:Rosewater", "flamingo:Flamingo", "pink:Pink", "mauve:Mauve", "red:Red", "maroon:Maroon", "peach:Peach", "yellow:Yellow", "green:Green", "teal:Teal", "blue:Blue", "sapphire:Sapphire*", "sky:Sky", "lavender:Lavender", "subtext0:Gray"]
+==/UserStyle== */
+
+@-moz-document url-prefix('https://www.roblox.com') {
```

Comment:
```suggestion
@-moz-document domain('roblox.com') {
```

Reply:
mkay

Reply:
Having ``domain('roblox.com')`` would result in Stylus believing the theme should be applied across all Roblox domains including ``devforum.roblox.com``, ``corp.roblox.com``, ``education.roblox.com`` and others. It does appear if I make it ``www.roblox.com`` the prior stated problem doesn't occur. So I'll switch it do that.

Reply:
Updated the suggestion.

Reply:
I've changed it to ``domain('www.roblox.com')`` on my end so once I push it I'll resolve the conversation instead of committing the suggestion (had to fix a few elements that were unstyled and decided to do this whilst at it.)

Reply:
It was there but it appears that somehow with my infinite intelligence I deleted it. I think it happened when I was updating the user style from a local version I have on Stylus and the Github file. Will fix it.